api/models.py

```diff
@@ -77,6 +77,18 @@ class Movie(models.Model):
     def __str__(self):
         return self.title
 
+    @property
+    def running_time(self):
+        schedule = Schedule.objects.filter(
```

Comment:
음 이부분은 에러가 날 것 같네요😢여기서 self는 Movie 모델의 인스턴스를 나타내기 때문에 `self.start_time`이나 `self.finish_time`은 존재하지 않을 거예요
movie와 연결된 schedule 쿼리셋을 가져오고 싶은거라면 다음과 같이 하시는게 좋을거 같아요 (그리고 가져오는 객체가 단수가 아니라 복수이기 때문에 `schedule`보다는 `schedules`가 나을 것 같아요!

```suggestion
        schedules = self.schedule
```
+ 여기서 self.schedule인 이유는, Schedule 모델의 movie 필드 설정 시에 `related_name`을 `schedule`로 설정해주셨기 때문이에요!